.nuspec/Microsoft.Maui.Controls.SingleProject.targets

```diff
@@ -60,5 +62,11 @@
   <ItemGroup Condition=" '$(SingleProject)' == 'true' ">
     <ProjectCapability Include="MauiSingleProject" />
     <ProjectCapability Include="LaunchProfiles" />
+    <!--
+      Optional - Enables a list of TFM's and device categories in the debug menu.
+      This allows easily toggling of debug target TFM by selecting the platform.
+      If removed, top level debug targets show as a list of devices for the selected TFM.
+      -->
+    <ProjectCapability Include="XamarinStaticLaunchProfiles" />
```

Comment:
/cc @joj @tondat this is probably the portion related to the IDE.

@tondat I tried to mark you as a reviewer, but maybe you don't have contributor access on this repo.

Reply:
@tondat is suggesting in Teams we might change to:
```xml
<ItemGroup>
  <ProjectCapability Include="MauiSingleProject" />
  <ProjectCapability Include="LaunchProfilesGroupByPlatformFilters"/>
  <ProjectCapability Include="BuildOnlyActiveTargetFrameworkOnDebug" />
</ItemGroup>
```
Also, a target that would improve the developer loop (until we get the appropriate Project System changes);
```xml
<Target Name="ComputeXamarinTargetFrameworks" BeforeTargets="_ComputeTargetFrameworkItems">
  <PropertyGroup>
    <TargetFrameworks Condition="'$(TargetFramework)' == '' AND '$(_SelectedTargetFramework)' != ''">$(_SelectedTargetFramework)</TargetFrameworks>
  </PropertyGroup>
  <Message Text="SelectedTargetFramework = $(_SelectedTargetFramework)" />
  <Message Text="TargetFramework = $(TargetFramework)" />
  <Message Text="TargetFrameworks = $(TargetFrameworks)" />
  <Message Text="BuildingInsideVisualStudio = $(BuildingInsideVisualStudio)" />
</Target>
```
I'll want to test this target a bit, and probably refactor it.

Reply:
Final capabilities might be:
```xml
<ItemGroup>
  <ProjectCapability Include="MauiSingleProject" />
  <ProjectCapability Include="XamarinStaticLaunchProfiles" Condition=" '$(VisualStudioVersion)' &lt; '17.0' " />
  <ProjectCapability Include="LaunchProfilesGroupByPlatformFilters" Condition=" '$(VisualStudioVersion)' &gt;= '17.0' " />
  <ProjectCapability Include="BuildOnlyActiveTargetFrameworkOnDebug" />
</ItemGroup>
```
`LaunchProfilesGroupByPlatformFilters` is Dev17-only, and you can't have `XamarinStaticLaunchProfiles` at the same time.